src/ensembl/src/content/app/entity-viewer/components/transcript-filter/EntityViewerTranscriptFilter.scss

```diff
@@ -0,0 +1,36 @@
+@import 'src/styles/common';
+
+$checkbox-border-color: rgba(154, 167, 177, 0.4);
+$checkbox-background-color: rgba(111, 129, 144, 0.3);
+$option-group-border-color: rgba(255, 153, 0, 0.4);
```

Comment:
Are these our regular colour (apart from the opacity)?

If not — why not? If yes, then we can use SCSS variables here (`$checkbox-border-color: rgba($grey, 0.4);`).

Reply:
I was not aware of that. It's done now. Thanks 👍